catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/downloadManager.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="Resource Download Configuration"
+         name="Resource Download Settings"
+         id="ddf.catalog.resource.download.ReliableResourceDownloadManager">
+        <AD name="Product Cache Directory" id="productCacheDirectory" required="false"
+            type="String" default=""
+            description="Directory where retrieved products will be cached for faster, future retrieval.
+             If a directory path is specified with directories that do not exist,
+             Catalog Framework will attempt to create those directories.
```

Comment:
Instead of _Catalog Framework_, should we say _the Product Download feature_ or something like that?